systemverilog-plugin/UhdmAst.cc

```diff
@@ -4547,6 +4607,73 @@ void UhdmAst::process_unsupported_stmt(const UHDM::BaseClass *object, bool is_er
     log_func("%sCurrently not supported object of type '%s'\n", prefix.c_str(), UHDM::VpiTypeName(obj_h).c_str());
 }
 
+void UhdmAst::process_type_parameter()
+{
+    current_node = make_ast_node(AST::AST_PARAMETER);
+    static int anonymous_type_id = 0;
+    static int enum_id = 0;
```

Comment:
Please add this static variables to ``uhdmastshared``. It also looks like there is already some static for enum: https://github.com/chipsalliance/yosys-f4pga-plugins/blob/main/systemverilog-plugin/uhdmastshared.h#L32

Can we use it?

Reply:
That's right, now we're using the existing enum counter, and the counter for anonymous types works analogously to what we've already had in `uhdmastshared`.